<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>861</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Candidate List</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>721</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1131</width>
         <height>647</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Candidates Managment</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>83</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>83</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Field</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>250</y>
          <width>131</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>MP</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>131</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>PC</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>50</y>
          <width>108</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>120</y>
          <width>108</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>190</y>
          <width>108</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1131</width>
         <height>647</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Available Candidates</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>-10</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 30pt &quot;Cantarell&quot;;</string>
        </property>
        <property name="text">
         <string>Student LIST</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>1111</width>
          <height>581</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <column>
         <property name="text">
          <string>CIN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>First Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Field</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>730</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Marks List</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>1111</width>
        <height>751</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>CIN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Analysis</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Algebra</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Physics</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chemistry</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer Science</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>French</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>English</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualize Marks Distribution</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Assign students</string>
     </attribute>
     <widget class="QToolBox" name="toolBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>801</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1071</width>
         <height>727</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Set DeadLine</string>
       </attribute>
       <widget class="QCalendarWidget" name="calendarWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1081</width>
          <height>631</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1071</width>
         <height>727</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Choices list</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>1081</width>
          <height>601</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>CIN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Choice 1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Choice 2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Choice 3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rank</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>650</y>
          <width>221</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Assign</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
